src/index.ts

```diff
@@ -0,0 +1,113 @@
+import { IFileData } from './types';
+import untarjs from '@emscripten-forge/untarjs';
+
+const checkConda = (url: string): boolean => {
+  let flag = false;
+  if (url.toLowerCase().endsWith('.conda')) {
+    flag = true;
+  }
+  return flag;
+};
+
+const installCondaPackage = async (
+  prefix: string,
+  url: string,
+  FS: any
+): Promise<void> => {
+  let files: IFileData[] = await untarjs.extract(url);
+  const isCondaPackage = checkConda(url);
+  if (isCondaPackage) {
+    console.log(`!!extract conda package ${url})`);
```

Comment:
Good for debugging, but we should not log anything. 

We could have an optional argument `verbose? = false` if you want though.